api/base/filters.py

```diff
@@ -24,20 +24,23 @@
 def sort_multiple(fields):
     fields = list(fields)
     def sort_fn(a, b):
-        while fields:
-            field = fields.pop(0)
+        sort_direction = 1
+        for index, field in enumerate(fields):
+            if field[0] == '-':
+                sort_direction = -1
+                field = field[1:]
             a_field = getattr(a, field)
             b_field = getattr(b, field)
             if a_field > b_field:
-                return 1
+                return 1 * sort_direction
             elif a_field < b_field:
-                return -1
+                return -1 * sort_direction
+            continue
```

Comment:
`continue` is unnecessary at the end of a loop. It would only be used if you wanted to skip some additional code in the block of the loop. When there's nothing to skip, it's unneeded. Let's remove it! :-)